### DISCUSSION TITLE
How to setup Community edition for Rocket Chat repository

### DISCUSSION THREAD

**Discussion by THAQuanttt** [RocketChat/Rocket.Chat#34053] [Q&A] [answered]
Hi guys, , I’m trying to self-host Rocket Chat and I have to remove Enterprise Edition from Rocket Chat’s source code to avoid violating the EE license. I forked RocketChat Repo from Github [ref](https://github.com/RocketChat/Rocket.Chat) and I ran the fossity.ts file in the scripts folder to remove apps/meteor/ee/ and ee/, however after running the Script, when I run the Yarn command, it gives that error. I checked in source code and then I noticed there are many files import "rocket.chat/license", "@rocket.chat/network-broker", "@rocket.chat/omnichannel-services",... which are deleted when running fossify script and I don't know what to do next!!

Please let me know what else I need to do after I fork the Rocket Chat repository and run the fossify.ts file to be able to run the Rocket Chat server.

I really thank you very much!!

**Server Setup Information:**
- Version of Rocket.Chat Server: 7.0.0
- License Type:
- Number of Users: 
- Operating System: ubuntu
- Deployment Method: development
- Number of Running Instances:
- DB Replicaset Oplog:
- NodeJS Version: 20.18.0
- MongoDB Version:

**Comment by namnguyenthanhwork** [comment]
@THAQuanttt  Have you solved this problem? I'm the same as you.

**Reply by THAQuanttt** [reply]
> @THAQuanttt Have you solved this problem? I'm the same as you.

ya, currently the fossify script only removes the EE code itself, so you will have to find and delete all references to EE yourself.

**Comment by reetp** [comment, marked as answer]
This was answered here

https://open.rocket.chat/channel/support?msg=q2Ko93XC3vrR7ipwa

There's a difference between Community Edition and the FOSS version;

Community Edition is the basic [rocket.chat](https://rocket.chat/) as it exists on the repository, just without any enterprise license applied to it. It can use the code that is flagged as EE;

The FOSS version on the other hand includes only the code under the MIT license. Currently there's no "ready to build" version of it available yet. The fossify script just removes everything that isn't FOSS,.

 > So if we run the fossify script and remove those references, along with any related functions and logic from those EE packages by ourself, still we are not violating the EE license?

Correct.
